src/test/kotlin/graphics/scenery/tests/examples/volumes/CroppingExample.kt

```diff
@@ -69,10 +69,12 @@ class CroppingExample : SceneryBase("Volume Cropping example", 1280, 720) {
         origin.material().diffuse = Vector3f(0.8f, 0.0f, 0.0f)
         scene.addChild(origin)
 
-        val imp: ImagePlus = IJ.openImage("https://imagej.nih.gov/ij/images/t1-head.zip")
-        val img: Img<UnsignedShortType> = ImageJFunctions.wrapShort(imp)
+        val resource = Volume.VolumeFileSource(
+            Volume.VolumeFileSource.VolumePath.Resource("/graphics/scenery/tests/unit/volume/t1-head.zip"),
```

Comment:
The idea here is good, but this needs to work without the whole ImageJ dependency.

Reply:
thanks. resolved